src/app/layout.tsx

```diff
@@ -7,21 +11,28 @@ import '@mantine/dropzone/styles.layer.css'
 
 import { Providers } from './providers'
 import { AppLayout } from '@/components/layout/app-layout'
-import { ReactNode } from 'react'
+import { type ReactNode } from 'react'
 
-export const metadata: Metadata = {
-    title: 'SafeInsights Management Application',
-    description: 'Manages studies, members, and data',
-    icons: {
-        icon: '/icon.png',
-    },
+export async function generateMetadata(): Promise<Metadata> {
```

Comment:
any reason to switch this other than stylistic? since this is all static data?

https://nextjs.org/docs/app/api-reference/functions/generate-metadata

Reply:
the file switched to server-rendered, and nexts complained that "only functions may be exported from server-rendered components"

Reply:
weird!! good to know